src/channel/index.js

```diff
@@ -0,0 +1,193 @@
+const uuid = require('uuid');
+
+class Channel {
+  constructor(logger) {
+    this.logger = logger;
+
+    /**
+     * Queue id
+     * @type {String}
+     * @public
+     */
+    this.qid = uuid.v4();
+
+    /**
+     * @type {String}
+     * @public
+     */
+    this.queueName = `rpc-node-response-${this.qid}`;
+
+    this.create = this.create.bind(this);
+    this.close = this.close.bind(this);
+    this.assertQueue = this.assertQueue.bind(this);
+    this.assertExchange = this.assertExchange.bind(this);
+    this.bindQueue = this.bindQueue.bind(this);
+    this.publish = this.publish.bind(this);
+    this.ack = this.ack.bind(this);
+  }
+
+  /**
+   * Sets up the queue and the exchange for AMQ
```

Comment:
```suggestion
   * Sets up the queue and the exchange in RabbitMQ
```
AMQP is the protocol, it knows nothing about setting up queues and exhanges.